examples/pruning/tensorflow_estimator_integration.py

```diff
@@ -24,90 +24,78 @@
 TRAIN_STEPS = 1000
 EVAL_STEPS = 100
```

Comment:
Please remove this line because `EVAL_STEPS` is not used anymore.
```suggestion
```

Reply:
Sorry, I missed it.
I removed `EVAL_STEPS`.